include/exadg/structure/spatial_discretization/operator.cpp

```diff
@@ -710,11 +718,14 @@ Operator<dim, Number>::compute_initial_acceleration(VectorType &       accelerat
   VectorType rhs(acceleration);
   rhs = 0.0;
 
+  double scaling_factor_mass;
+
```

Comment:
we set the factor to zero, so store the value we had before and reset it after computing the initial acceleration